scripts/upsify-daq-pkgs.py

```diff
@@ -7,6 +7,7 @@
 import getpass
 import datetime
 import argparse
+import os.path as path
```

Comment:
`import os` exists before this. Some style guide recommends not to use `import ... as ...`; but instead use `from ... import ...`.

I suggest change this line to `from os import path`; and remove the previous `import os`; or remove this line and replace `path` with `os.path` in the context of the script.